examples/hosts/app-integration/external-data/src/mock-customer-service/service.ts

```diff
@@ -163,6 +190,58 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 		}
 	});
 
+	/**
+	 * "Echoes" the external data services data update notifications to our own webhook subscribers.
+	 *
+	 * Expected input data format:
+	 *
+	 * ```json
+	 *	{
+	 *		taskList: {
+	 *			[id: string]: {
+	 *				name: string,
+	 *				priority: number
+	 *			}
+	 *		}
+	 *	}
+	 * ```
+	 *
+	 * This data will be forwarded to our own subscribers.
+	 */
+	expressApp.post("/register-session-url", (request, result) => {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		const containerUrl = request.body?.containerUrl as string;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		const externalTaskListId = request.body?.externalTaskListId as string;
+		if (containerUrl === undefined) {
+			const errorMessage =
+				'No session data provided by client. Expected under "sessionUrl" property.';
+			console.error(formatLogMessage(errorMessage));
+			result.status(400).json({ message: errorMessage });
+		} else {
+			clientManager.registerClient(containerUrl, externalTaskListId);
+			console.log(
+				formatLogMessage(
+					`Registered containerUrl ${containerUrl} with external query: ${externalTaskListId}".`,
+				),
+			);
+			registerForWebhook(
```

Comment:
This probably wants to be conditional on us not having a subscription for this external task list id on the external data service already. Otherwise the customer service might get called several times for the same update, and in turn send more notifications than necessary to the clients.